tests/Hedgehog.Tests/TreeTests.fs

```diff
@@ -0,0 +1,58 @@
+﻿module Hedgehog.Tests.TreeTests
+
+open Hedgehog
+open Swensen.Unquote
+open Xunit
+
+[<Fact>]
+let ``render of singleton tree`` () =
+    Property.check <| property {
+        let! v = Gen.uint64 (Range.exponentialBounded ())
```

Comment:
Any strong reason around `uint64` instead of something smaller? e.g. `uint16`

Reply:
I just copied it from some other test.  I think any generator will do here.